documentation/design-docs/dotnet-tools.md

```diff
@@ -208,24 +196,17 @@ MONITOR
 
 
     Examples:
-      > dotnet counters monitor --processId 1902 System.Runtime Microsoft.AspNet
-
+      > dotnet-counters monitor --process-id 1902 --refresh-interval 3 System.Runtime 
```

Comment:
I know it says in the help text what the interval units are, could you include a description of what this invocation is doing here as well?